ci/Dockerfile

```diff
@@ -22,8 +22,12 @@ RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
 RUN mkdir eclipse-ltp \
     && cd eclipse-ltp \
     && git clone https://github.com/OpenLiberty/liberty-tools-eclipse.git
-
+ 
 WORKDIR /eclipse-ltp/liberty-tools-eclipse
+RUN git checkout repo-loc
```

Comment:
Did you mean to delete this?

Reply:
yes, that should have been removed, done